app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -159,4 +154,33 @@ public function getStoreIds()
         return $storeIds;
     }
 
+    private function getOrderProducts(Mage_Sales_Model_Order $order): array {
+        $products = [];
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            $products[] = [
+                'name' => $product->getName(),
+                'url' => $product->getProductUrl(),
+                'id' => $product->getEntityId(),
+                'sku' => $product->getSku(),
+                'image_url' => '',
+                'brand' => $product->getAttributeText('manufacturer'),
+            ];
+        }
+
+        return $products;
+    }
+
+    private function getLanguage(int $storeId, Mage_Sales_Model_Order $order): string {
+        $language = Mage::getStoreConfig('webwinkelconnect/invitation/language', $storeId);
+        if (empty($language) || $language == '') {
+            return explode('_', Mage::getStoreConfig('general/locale/code',$storeId))[1];
```

Comment:
Could you provide an example of what is ` Mage::getStoreConfig('general/locale/code'` and what would be the return of `getLanguage`

Reply:
If a language is chosen in the config, it will return e.g. "bul"
If the language is set to "based on customer country" it will return the country code in lower case e.g. "bg"
If the language is set to "use default" it will return the store language code in lower case e.g. "bg"
Bear in mind that this will be the function result from the upcoming commit.